CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+# dbt_recurly v0.5.0
+[PR #29](https://github.com/fivetran/dbt_recurly/pull/29) includes the following breaking changes:
+## Breaking Changes
+- Removed unused fields from `stg_recurly__subscription_history` in the upstream `dbt_recurly_source` package (see [CHANGELOG](https://github.com/fivetran/dbt_recurly_source/blob/main/CHANGELOG.md#dbt_recurly_source-v030)):
```

Comment:
Small suggestion to link the release notes instead of the CHANGELOG. This way it directs the customer to a focused version of the CHANGELOG in the form of the release.

Reply:
Updated.